### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by JordanMartinez**
Backlinking to purescript/purescript#3942

### CODE REVIEW

**Review by kl0tl — COMMENTED** [review]
Shouldn’t we also upgrade the bower dependencies to master?

File: spago.dhall [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{-
+Welcome to a Spago project!
+You can edit this file as you like.
+-}
```
Comment (kl0tl) [inline spago.dhall]: Shall we remove this comment?

**Comment by JordanMartinez** [comment]
> Shouldn’t we also upgrade the bower dependencies to master?

Hm... I guess it doesn't hurt. Who knows. It might be required for those who will still decide to use bower when they update their library?

**Comment by thomashoneyman** [comment]
> Shouldn’t we also upgrade the bower dependencies to master?

Yea, we should upgrade the bower dependencies to master for all of these `-web` updates. Fortunately, it looks like that's been done on almost all the other PRs already.

File: spago.dhall (hunk 1 above)
Comment (JordanMartinez) [reply spago.dhall]: Done.

**Review by thomashoneyman — APPROVED** [review]

**Comment by JordanMartinez** [comment]
Just waiting for CI now.

**Comment by hdgarrood** [comment]
I think the same reasoning for only including bower.json in the core libraries applies here: I don't think we gain much from having both bower and spago configurations in the repo. It just means there's two workflows we need to maintain rather than one. We shouldn't keep two sets of dependencies in the repo if we're not maintaining them both, so if we want to maintain both bower and spago configurations here then we should ensure both of them are tested by CI. However, you don't need to have a spago.dhall in a package's repo in order to consume that package with spago, so from my point of view I think it's preferable to only support bower for people working on the package.

**Comment by hdgarrood** [comment]
Let's have this discussion on https://github.com/purescript/purescript/issues/3942

**Comment by JordanMartinez** [comment]
Sounds good. I'll wait for you to post there.

**Pull request closed without merging**